microk8s-resources/default-hooks/reconcile.d/10-cni-restart

```diff
@@ -29,3 +23,12 @@ then
     rm "${SNAP_DATA}/var/lock/cni-needs-reload"
   fi
 fi
+
+if  ! [ -e "${SNAP_DATA}/var/lock/skip-offloading" ] &&
+    [ -e "${SNAP_DATA}/var/lock/try-offloading-patch" ] &&
```

Comment:
How about this?
```suggestion
    ! [ -e "${SNAP_DATA}/var/lock/offloading-patch-applied" ] &&
```

Then you can also skip the `try-offloading-patch` entirely.